packages/list/ListItemText.tsx

```diff
@@ -45,15 +58,23 @@ const ListItemText = (props) => {
         </span>
       );
     }
+    if (!isReactElement(text)) return null;
+
     const {className: textClassName, ...otherProps} = text.props;
-    const props = Object.assign({
-      className: classnames(className, textClassName),
-    }, ...otherProps);
+    const props = Object.assign(
```

Comment:
This could be the spread operator instead of Object.assign. Also, there was something similar above that passed these properties in a slightly different order. The objects passed in later in the args list will overwrite any duplicate properties from previous args

Reply:
you're talking about this block? About overwriting?

```
const ListItemText: React.FunctionComponent<ListItemTextProps> = ({
  /* eslint-disable react/prop-types */
  primaryText = '',
  secondaryText = '',
  tabbableOnListItemFocus = false,
  tabIndex = -1,
  className = '',
  /* eslint-enable react/prop-types */
  ...otherProps
}) => {```